CRITICAL - 2024-07-15 16:21:57 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 246.
 1 SYSTEMPATH\CodeIgniter.php(246): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(210): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 FCPATH\index.php(67): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2024-07-15 16:21:58 --> Uncaught Error: Class "Locale" not found in C:\xampp\htdocs\machine_Test\system\Helpers\number_helper.php:136
Stack trace:
#0 C:\xampp\htdocs\machine_Test\system\Helpers\number_helper.php(55): format_number(4.0, 2, NULL, Array)
#1 C:\xampp\htdocs\machine_Test\system\Debug\BaseExceptionHandler.php(145): number_to_size(4.0, 2)
#2 C:\xampp\htdocs\machine_Test\app\Views\errors\html\error_exception.php(360): CodeIgniter\Debug\BaseExceptionHandler::describeMemory(4194304)
#3 C:\xampp\htdocs\machine_Test\system\Debug\BaseExceptionHandler.php(241): include('C:\\xampp\\htdocs...')
#4 C:\xampp\htdocs\machine_Test\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#5 C:\xampp\htdocs\machine_Test\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(CodeIgniter\Exceptions\FrameworkException), 500, 'C:\\xampp\\htdocs...')
#6 C:\xampp\htdocs\machine_Test\system\Debug\Exceptions.php(146): CodeIgniter\Debug\ExceptionHandler->handle(Object(CodeIgniter\Exceptions\FrameworkException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#7 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Exceptions\FrameworkException))
#8 {main}
  thrown
【Previous Exception】
CodeIgniter\Exceptions\FrameworkException
The framework needs the following extension(s) installed and loaded: intl.
#0 C:\xampp\htdocs\machine_Test\system\CodeIgniter.php(246): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
#1 C:\xampp\htdocs\machine_Test\system\CodeIgniter.php(210): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
#2 C:\xampp\htdocs\machine_Test\index.php(67): CodeIgniter\CodeIgniter->initialize()
#3 {main}
in SYSTEMPATH\Helpers\number_helper.php on line 136.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-07-15 16:22:06 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 246.
 1 SYSTEMPATH\CodeIgniter.php(246): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(210): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 FCPATH\index.php(67): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2024-07-15 16:22:07 --> Uncaught Error: Class "Locale" not found in C:\xampp\htdocs\machine_Test\system\Helpers\number_helper.php:136
Stack trace:
#0 C:\xampp\htdocs\machine_Test\system\Helpers\number_helper.php(55): format_number(4.0, 2, NULL, Array)
#1 C:\xampp\htdocs\machine_Test\system\Debug\BaseExceptionHandler.php(145): number_to_size(4.0, 2)
#2 C:\xampp\htdocs\machine_Test\app\Views\errors\html\error_exception.php(360): CodeIgniter\Debug\BaseExceptionHandler::describeMemory(4194304)
#3 C:\xampp\htdocs\machine_Test\system\Debug\BaseExceptionHandler.php(241): include('C:\\xampp\\htdocs...')
#4 C:\xampp\htdocs\machine_Test\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#5 C:\xampp\htdocs\machine_Test\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(CodeIgniter\Exceptions\FrameworkException), 500, 'C:\\xampp\\htdocs...')
#6 C:\xampp\htdocs\machine_Test\system\Debug\Exceptions.php(146): CodeIgniter\Debug\ExceptionHandler->handle(Object(CodeIgniter\Exceptions\FrameworkException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#7 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Exceptions\FrameworkException))
#8 {main}
  thrown
【Previous Exception】
CodeIgniter\Exceptions\FrameworkException
The framework needs the following extension(s) installed and loaded: intl.
#0 C:\xampp\htdocs\machine_Test\system\CodeIgniter.php(246): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
#1 C:\xampp\htdocs\machine_Test\system\CodeIgniter.php(210): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
#2 C:\xampp\htdocs\machine_Test\index.php(67): CodeIgniter\CodeIgniter->initialize()
#3 {main}
in SYSTEMPATH\Helpers\number_helper.php on line 136.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-07-15 16:22:39 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 246.
 1 SYSTEMPATH\CodeIgniter.php(246): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(210): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 FCPATH\index.php(67): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2024-07-15 16:22:39 --> Uncaught Error: Class "Locale" not found in C:\xampp\htdocs\machine_Test\system\Helpers\number_helper.php:136
Stack trace:
#0 C:\xampp\htdocs\machine_Test\system\Helpers\number_helper.php(55): format_number(4.0, 2, NULL, Array)
#1 C:\xampp\htdocs\machine_Test\system\Debug\BaseExceptionHandler.php(145): number_to_size(4.0, 2)
#2 C:\xampp\htdocs\machine_Test\app\Views\errors\html\error_exception.php(360): CodeIgniter\Debug\BaseExceptionHandler::describeMemory(4194304)
#3 C:\xampp\htdocs\machine_Test\system\Debug\BaseExceptionHandler.php(241): include('C:\\xampp\\htdocs...')
#4 C:\xampp\htdocs\machine_Test\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#5 C:\xampp\htdocs\machine_Test\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(CodeIgniter\Exceptions\FrameworkException), 500, 'C:\\xampp\\htdocs...')
#6 C:\xampp\htdocs\machine_Test\system\Debug\Exceptions.php(146): CodeIgniter\Debug\ExceptionHandler->handle(Object(CodeIgniter\Exceptions\FrameworkException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#7 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Exceptions\FrameworkException))
#8 {main}
  thrown
【Previous Exception】
CodeIgniter\Exceptions\FrameworkException
The framework needs the following extension(s) installed and loaded: intl.
#0 C:\xampp\htdocs\machine_Test\system\CodeIgniter.php(246): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
#1 C:\xampp\htdocs\machine_Test\system\CodeIgniter.php(210): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
#2 C:\xampp\htdocs\machine_Test\index.php(67): CodeIgniter\CodeIgniter->initialize()
#3 {main}
in SYSTEMPATH\Helpers\number_helper.php on line 136.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-07-15 16:22:47 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 246.
 1 SYSTEMPATH\CodeIgniter.php(246): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(210): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 FCPATH\index.php(67): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2024-07-15 16:22:47 --> Uncaught Error: Class "Locale" not found in C:\xampp\htdocs\machine_Test\system\Helpers\number_helper.php:136
Stack trace:
#0 C:\xampp\htdocs\machine_Test\system\Helpers\number_helper.php(55): format_number(4.0, 2, NULL, Array)
#1 C:\xampp\htdocs\machine_Test\system\Debug\BaseExceptionHandler.php(145): number_to_size(4.0, 2)
#2 C:\xampp\htdocs\machine_Test\app\Views\errors\html\error_exception.php(360): CodeIgniter\Debug\BaseExceptionHandler::describeMemory(4194304)
#3 C:\xampp\htdocs\machine_Test\system\Debug\BaseExceptionHandler.php(241): include('C:\\xampp\\htdocs...')
#4 C:\xampp\htdocs\machine_Test\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#5 C:\xampp\htdocs\machine_Test\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(CodeIgniter\Exceptions\FrameworkException), 500, 'C:\\xampp\\htdocs...')
#6 C:\xampp\htdocs\machine_Test\system\Debug\Exceptions.php(146): CodeIgniter\Debug\ExceptionHandler->handle(Object(CodeIgniter\Exceptions\FrameworkException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#7 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Exceptions\FrameworkException))
#8 {main}
  thrown
【Previous Exception】
CodeIgniter\Exceptions\FrameworkException
The framework needs the following extension(s) installed and loaded: intl.
#0 C:\xampp\htdocs\machine_Test\system\CodeIgniter.php(246): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
#1 C:\xampp\htdocs\machine_Test\system\CodeIgniter.php(210): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
#2 C:\xampp\htdocs\machine_Test\index.php(67): CodeIgniter\CodeIgniter->initialize()
#3 {main}
in SYSTEMPATH\Helpers\number_helper.php on line 136.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-07-15 16:22:47 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 246.
 1 SYSTEMPATH\CodeIgniter.php(246): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(210): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 FCPATH\index.php(67): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2024-07-15 16:22:47 --> Uncaught Error: Class "Locale" not found in C:\xampp\htdocs\machine_Test\system\Helpers\number_helper.php:136
Stack trace:
#0 C:\xampp\htdocs\machine_Test\system\Helpers\number_helper.php(55): format_number(4.0, 2, NULL, Array)
#1 C:\xampp\htdocs\machine_Test\system\Debug\BaseExceptionHandler.php(145): number_to_size(4.0, 2)
#2 C:\xampp\htdocs\machine_Test\app\Views\errors\html\error_exception.php(360): CodeIgniter\Debug\BaseExceptionHandler::describeMemory(4194304)
#3 C:\xampp\htdocs\machine_Test\system\Debug\BaseExceptionHandler.php(241): include('C:\\xampp\\htdocs...')
#4 C:\xampp\htdocs\machine_Test\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#5 C:\xampp\htdocs\machine_Test\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(CodeIgniter\Exceptions\FrameworkException), 500, 'C:\\xampp\\htdocs...')
#6 C:\xampp\htdocs\machine_Test\system\Debug\Exceptions.php(146): CodeIgniter\Debug\ExceptionHandler->handle(Object(CodeIgniter\Exceptions\FrameworkException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#7 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Exceptions\FrameworkException))
#8 {main}
  thrown
【Previous Exception】
CodeIgniter\Exceptions\FrameworkException
The framework needs the following extension(s) installed and loaded: intl.
#0 C:\xampp\htdocs\machine_Test\system\CodeIgniter.php(246): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
#1 C:\xampp\htdocs\machine_Test\system\CodeIgniter.php(210): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
#2 C:\xampp\htdocs\machine_Test\index.php(67): CodeIgniter\CodeIgniter->initialize()
#3 {main}
in SYSTEMPATH\Helpers\number_helper.php on line 136.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-07-15 16:25:35 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 246.
 1 SYSTEMPATH\CodeIgniter.php(246): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(210): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 FCPATH\index.php(67): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2024-07-15 16:25:35 --> Uncaught Error: Class "Locale" not found in C:\xampp\htdocs\machine_Test\system\Helpers\number_helper.php:136
Stack trace:
#0 C:\xampp\htdocs\machine_Test\system\Helpers\number_helper.php(55): format_number(4.0, 2, NULL, Array)
#1 C:\xampp\htdocs\machine_Test\system\Debug\BaseExceptionHandler.php(145): number_to_size(4.0, 2)
#2 C:\xampp\htdocs\machine_Test\app\Views\errors\html\error_exception.php(360): CodeIgniter\Debug\BaseExceptionHandler::describeMemory(4194304)
#3 C:\xampp\htdocs\machine_Test\system\Debug\BaseExceptionHandler.php(241): include('C:\\xampp\\htdocs...')
#4 C:\xampp\htdocs\machine_Test\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#5 C:\xampp\htdocs\machine_Test\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(CodeIgniter\Exceptions\FrameworkException), 500, 'C:\\xampp\\htdocs...')
#6 C:\xampp\htdocs\machine_Test\system\Debug\Exceptions.php(146): CodeIgniter\Debug\ExceptionHandler->handle(Object(CodeIgniter\Exceptions\FrameworkException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#7 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Exceptions\FrameworkException))
#8 {main}
  thrown
【Previous Exception】
CodeIgniter\Exceptions\FrameworkException
The framework needs the following extension(s) installed and loaded: intl.
#0 C:\xampp\htdocs\machine_Test\system\CodeIgniter.php(246): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
#1 C:\xampp\htdocs\machine_Test\system\CodeIgniter.php(210): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
#2 C:\xampp\htdocs\machine_Test\index.php(67): CodeIgniter\CodeIgniter->initialize()
#3 {main}
in SYSTEMPATH\Helpers\number_helper.php on line 136.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-07-15 16:25:38 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 246.
 1 SYSTEMPATH\CodeIgniter.php(246): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(210): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 FCPATH\index.php(67): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2024-07-15 16:25:38 --> Uncaught Error: Class "Locale" not found in C:\xampp\htdocs\machine_Test\system\Helpers\number_helper.php:136
Stack trace:
#0 C:\xampp\htdocs\machine_Test\system\Helpers\number_helper.php(55): format_number(4.0, 2, NULL, Array)
#1 C:\xampp\htdocs\machine_Test\system\Debug\BaseExceptionHandler.php(145): number_to_size(4.0, 2)
#2 C:\xampp\htdocs\machine_Test\app\Views\errors\html\error_exception.php(360): CodeIgniter\Debug\BaseExceptionHandler::describeMemory(4194304)
#3 C:\xampp\htdocs\machine_Test\system\Debug\BaseExceptionHandler.php(241): include('C:\\xampp\\htdocs...')
#4 C:\xampp\htdocs\machine_Test\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#5 C:\xampp\htdocs\machine_Test\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(CodeIgniter\Exceptions\FrameworkException), 500, 'C:\\xampp\\htdocs...')
#6 C:\xampp\htdocs\machine_Test\system\Debug\Exceptions.php(146): CodeIgniter\Debug\ExceptionHandler->handle(Object(CodeIgniter\Exceptions\FrameworkException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#7 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Exceptions\FrameworkException))
#8 {main}
  thrown
【Previous Exception】
CodeIgniter\Exceptions\FrameworkException
The framework needs the following extension(s) installed and loaded: intl.
#0 C:\xampp\htdocs\machine_Test\system\CodeIgniter.php(246): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
#1 C:\xampp\htdocs\machine_Test\system\CodeIgniter.php(210): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
#2 C:\xampp\htdocs\machine_Test\index.php(67): CodeIgniter\CodeIgniter->initialize()
#3 {main}
in SYSTEMPATH\Helpers\number_helper.php on line 136.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-07-15 16:25:39 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 246.
 1 SYSTEMPATH\CodeIgniter.php(246): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(210): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 FCPATH\index.php(67): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2024-07-15 16:25:39 --> Uncaught Error: Class "Locale" not found in C:\xampp\htdocs\machine_Test\system\Helpers\number_helper.php:136
Stack trace:
#0 C:\xampp\htdocs\machine_Test\system\Helpers\number_helper.php(55): format_number(4.0, 2, NULL, Array)
#1 C:\xampp\htdocs\machine_Test\system\Debug\BaseExceptionHandler.php(145): number_to_size(4.0, 2)
#2 C:\xampp\htdocs\machine_Test\app\Views\errors\html\error_exception.php(360): CodeIgniter\Debug\BaseExceptionHandler::describeMemory(4194304)
#3 C:\xampp\htdocs\machine_Test\system\Debug\BaseExceptionHandler.php(241): include('C:\\xampp\\htdocs...')
#4 C:\xampp\htdocs\machine_Test\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#5 C:\xampp\htdocs\machine_Test\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(CodeIgniter\Exceptions\FrameworkException), 500, 'C:\\xampp\\htdocs...')
#6 C:\xampp\htdocs\machine_Test\system\Debug\Exceptions.php(146): CodeIgniter\Debug\ExceptionHandler->handle(Object(CodeIgniter\Exceptions\FrameworkException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#7 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Exceptions\FrameworkException))
#8 {main}
  thrown
【Previous Exception】
CodeIgniter\Exceptions\FrameworkException
The framework needs the following extension(s) installed and loaded: intl.
#0 C:\xampp\htdocs\machine_Test\system\CodeIgniter.php(246): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
#1 C:\xampp\htdocs\machine_Test\system\CodeIgniter.php(210): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
#2 C:\xampp\htdocs\machine_Test\index.php(67): CodeIgniter\CodeIgniter->initialize()
#3 {main}
in SYSTEMPATH\Helpers\number_helper.php on line 136.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-07-15 16:25:39 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 246.
 1 SYSTEMPATH\CodeIgniter.php(246): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(210): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 FCPATH\index.php(67): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2024-07-15 16:25:39 --> Uncaught Error: Class "Locale" not found in C:\xampp\htdocs\machine_Test\system\Helpers\number_helper.php:136
Stack trace:
#0 C:\xampp\htdocs\machine_Test\system\Helpers\number_helper.php(55): format_number(4.0, 2, NULL, Array)
#1 C:\xampp\htdocs\machine_Test\system\Debug\BaseExceptionHandler.php(145): number_to_size(4.0, 2)
#2 C:\xampp\htdocs\machine_Test\app\Views\errors\html\error_exception.php(360): CodeIgniter\Debug\BaseExceptionHandler::describeMemory(4194304)
#3 C:\xampp\htdocs\machine_Test\system\Debug\BaseExceptionHandler.php(241): include('C:\\xampp\\htdocs...')
#4 C:\xampp\htdocs\machine_Test\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#5 C:\xampp\htdocs\machine_Test\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(CodeIgniter\Exceptions\FrameworkException), 500, 'C:\\xampp\\htdocs...')
#6 C:\xampp\htdocs\machine_Test\system\Debug\Exceptions.php(146): CodeIgniter\Debug\ExceptionHandler->handle(Object(CodeIgniter\Exceptions\FrameworkException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#7 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Exceptions\FrameworkException))
#8 {main}
  thrown
【Previous Exception】
CodeIgniter\Exceptions\FrameworkException
The framework needs the following extension(s) installed and loaded: intl.
#0 C:\xampp\htdocs\machine_Test\system\CodeIgniter.php(246): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
#1 C:\xampp\htdocs\machine_Test\system\CodeIgniter.php(210): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
#2 C:\xampp\htdocs\machine_Test\index.php(67): CodeIgniter\CodeIgniter->initialize()
#3 {main}
in SYSTEMPATH\Helpers\number_helper.php on line 136.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-07-15 16:25:39 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 246.
 1 SYSTEMPATH\CodeIgniter.php(246): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(210): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 FCPATH\index.php(67): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2024-07-15 16:25:39 --> Uncaught Error: Class "Locale" not found in C:\xampp\htdocs\machine_Test\system\Helpers\number_helper.php:136
Stack trace:
#0 C:\xampp\htdocs\machine_Test\system\Helpers\number_helper.php(55): format_number(4.0, 2, NULL, Array)
#1 C:\xampp\htdocs\machine_Test\system\Debug\BaseExceptionHandler.php(145): number_to_size(4.0, 2)
#2 C:\xampp\htdocs\machine_Test\app\Views\errors\html\error_exception.php(360): CodeIgniter\Debug\BaseExceptionHandler::describeMemory(4194304)
#3 C:\xampp\htdocs\machine_Test\system\Debug\BaseExceptionHandler.php(241): include('C:\\xampp\\htdocs...')
#4 C:\xampp\htdocs\machine_Test\system\Debug\BaseExceptionHandler.php(244): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#5 C:\xampp\htdocs\machine_Test\system\Debug\ExceptionHandler.php(109): CodeIgniter\Debug\BaseExceptionHandler->render(Object(CodeIgniter\Exceptions\FrameworkException), 500, 'C:\\xampp\\htdocs...')
#6 C:\xampp\htdocs\machine_Test\system\Debug\Exceptions.php(146): CodeIgniter\Debug\ExceptionHandler->handle(Object(CodeIgniter\Exceptions\FrameworkException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#7 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Exceptions\FrameworkException))
#8 {main}
  thrown
【Previous Exception】
CodeIgniter\Exceptions\FrameworkException
The framework needs the following extension(s) installed and loaded: intl.
#0 C:\xampp\htdocs\machine_Test\system\CodeIgniter.php(246): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
#1 C:\xampp\htdocs\machine_Test\system\CodeIgniter.php(210): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
#2 C:\xampp\htdocs\machine_Test\index.php(67): CodeIgniter\CodeIgniter->initialize()
#3 {main}
in SYSTEMPATH\Helpers\number_helper.php on line 136.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
ERROR - 2024-07-15 14:38:55 --> mysqli_sql_exception: Table 'users' already exists in C:\xampp\htdocs\machine_Test\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\machine_Test\system\Database\MySQLi\Connection.php(306): mysqli->query('CREATE TABLE `u...', 0)
#1 C:\xampp\htdocs\machine_Test\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `u...')
#2 C:\xampp\htdocs\machine_Test\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `u...')
#3 C:\xampp\htdocs\machine_Test\system\Database\Forge.php(546): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `u...')
#4 C:\xampp\htdocs\machine_Test\app\Database\Migrations\2024-07-15-143808_CreateUserTable.php(48): CodeIgniter\Database\Forge->createTable('users')
#5 C:\xampp\htdocs\machine_Test\system\Database\MigrationRunner.php(877): App\Database\Migrations\CreateUserTable->up()
#6 C:\xampp\htdocs\machine_Test\system\Database\MigrationRunner.php(188): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\machine_Test\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\machine_Test\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\machine_Test\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\machine_Test\spark(102): CodeIgniter\CLI\Console->run()
#11 {main}
ERROR - 2024-07-15 15:21:34 --> mysqli_sql_exception: Unknown column 'created_at' in 'field list' in C:\xampp\htdocs\machine_Test\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\machine_Test\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\machine_Test\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\machine_Test\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\machine_Test\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\machine_Test\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\machine_Test\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\machine_Test\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\machine_Test\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\machine_Test\app\Controllers\User.php(45): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\machine_Test\system\CodeIgniter.php(919): App\Controllers\User->store()
#10 C:\xampp\htdocs\machine_Test\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#11 C:\xampp\htdocs\machine_Test\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\machine_Test\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\machine_Test\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-07-15 15:21:34 --> Unknown column 'created_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`name`, `email`, `mobile`, `profile_pic`, `password`, `created_at`, `updated_at`) VALUES (:name:, :email:, :mobile:, :profile_pic:, :password:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(698): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\User.php(45): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\User->store()
 8 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 9 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\machine_Test\\public\\index.php')
ERROR - 2024-07-15 17:36:56 --> mysqli_sql_exception: Unknown column 'created_at' in 'field list' in C:\xampp\htdocs\machine_Test\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\machine_Test\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\machine_Test\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\machine_Test\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\machine_Test\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\machine_Test\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\machine_Test\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\machine_Test\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\machine_Test\system\BaseModel.php(698): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\machine_Test\app\Controllers\User.php(45): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\machine_Test\system\CodeIgniter.php(919): App\Controllers\User->store()
#10 C:\xampp\htdocs\machine_Test\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#11 C:\xampp\htdocs\machine_Test\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\machine_Test\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\machine_Test\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-07-15 17:36:56 --> Unknown column 'created_at' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`name`, `email`, `mobile`, `profile_pic`, `password`, `created_at`, `updated_at`) VALUES (:name:, :email:, :mobile:, :profile_pic:, :password:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(698): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\User.php(45): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\User->store()
 8 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 9 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\machine_Test\\public\\index.php')
CRITICAL - 2024-07-15 17:45:22 --> Cannot declare class App\Database\Migrations\CreateUserTable, because the name is already in use
in APPPATH\Database\Migrations\2024-07-15-174513_CreateUserTable.php on line 7.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-07-15 17:46:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 18:56:59 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 19:30:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 19:30:59 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 19:31:03 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 19:33:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 19:34:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 19:37:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 19:37:38 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 19:38:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 19:38:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 19:40:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 19:41:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 19:41:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 19:43:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 19:43:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 19:45:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 19:46:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 19:49:06 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 19:49:21 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 19:54:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 19:55:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 19:55:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 19:55:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 19:55:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 20:03:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 20:06:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 20:06:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-07-15 20:08:34 --> Trying to access array offset on value of type null
in APPPATH\Views\user_edit.php on line 16.
 1 APPPATH\Views\user_edit.php(16): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp\\htdocs\\machine_Test\\app\\Views\\user_edit.php', 16)
 2 SYSTEMPATH\View\View.php(226): include('C:\\xampp\\htdocs\\machine_Test\\app\\Views\\user_edit.php')
 3 SYSTEMPATH\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1188): CodeIgniter\View\View->render('user_edit', [], true)
 5 APPPATH\Controllers\User.php(151): view('user_edit', [...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\User->update('2')
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\machine_Test\\public\\index.php')
INFO - 2024-07-15 20:08:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 20:10:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 20:11:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 20:13:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-07-15 20:17:51 --> {"password":"The password field is required."}
INFO - 2024-07-15 20:22:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 20:22:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 20:38:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 20:48:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 20:51:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 20:55:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 20:57:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 21:01:21 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 21:03:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 21:04:13 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 21:05:52 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 21:10:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 21:14:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 21:14:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-07-15 21:14:41 --> Config\App::$baseURL is invalid.
in SYSTEMPATH\HTTP\SiteURI.php on line 197.
 1 SYSTEMPATH\HTTP\SiteURI.php(143): CodeIgniter\HTTP\SiteURI->normalizeBaseURL(Object(Config\App))
 2 SYSTEMPATH\HTTP\SiteURI.php(98): CodeIgniter\HTTP\SiteURI->determineBaseURL(Object(Config\App), null, null)
 3 SYSTEMPATH\HTTP\SiteURIFactory.php(222): CodeIgniter\HTTP\SiteURI->__construct(Object(Config\App), 'create/', null)
 4 SYSTEMPATH\HTTP\SiteURIFactory.php(43): CodeIgniter\HTTP\SiteURIFactory->createURIFromRoutePath('create/')
 5 SYSTEMPATH\Config\Services.php(798): CodeIgniter\HTTP\SiteURIFactory->createFromGlobals()
 6 SYSTEMPATH\Config\BaseService.php(258): CodeIgniter\Config\Services::uri(null, false)
 7 SYSTEMPATH\Config\BaseService.php(199): CodeIgniter\Config\BaseService::__callStatic('uri', [...])
 8 SYSTEMPATH\Config\Services.php(791): CodeIgniter\Config\BaseService::getSharedInstance('uri', null)
 9 SYSTEMPATH\Config\BaseService.php(258): CodeIgniter\Config\Services::uri()
10 SYSTEMPATH\Config\Services.php(565): CodeIgniter\Config\BaseService::__callStatic('uri', [])
11 SYSTEMPATH\Config\Services.php(523): CodeIgniter\Config\Services::incomingrequest(Object(Config\App), false)
12 SYSTEMPATH\Config\BaseService.php(258): CodeIgniter\Config\Services::request(Object(Config\App), false)
13 SYSTEMPATH\Config\BaseService.php(199): CodeIgniter\Config\BaseService::__callStatic('request', [...])
14 SYSTEMPATH\Config\Services.php(558): CodeIgniter\Config\BaseService::getSharedInstance('request', Object(Config\App))
15 SYSTEMPATH\Config\BaseService.php(258): CodeIgniter\Config\Services::incomingrequest(Object(Config\App))
16 SYSTEMPATH\Config\Services.php(538): CodeIgniter\Config\BaseService::__callStatic('incomingrequest', [...])
17 SYSTEMPATH\Config\BaseService.php(258): CodeIgniter\Config\Services::createRequest(Object(Config\App))
18 SYSTEMPATH\CodeIgniter.php(639): CodeIgniter\Config\BaseService::__callStatic('createRequest', [...])
19 SYSTEMPATH\CodeIgniter.php(347): CodeIgniter\CodeIgniter->getRequestObject()
20 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
21 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\machine_Test\\public\\index.php')
CRITICAL - 2024-07-15 21:14:41 --> Uncaught CodeIgniter\Exceptions\ConfigException: Config\App::$baseURL is invalid. in C:\xampp\htdocs\machine_Test\system\HTTP\SiteURI.php:197
Stack trace:
#0 C:\xampp\htdocs\machine_Test\system\HTTP\SiteURI.php(143): CodeIgniter\HTTP\SiteURI->normalizeBaseURL(Object(Config\App))
#1 C:\xampp\htdocs\machine_Test\system\HTTP\SiteURI.php(98): CodeIgniter\HTTP\SiteURI->determineBaseURL(Object(Config\App), NULL, NULL)
#2 C:\xampp\htdocs\machine_Test\system\HTTP\SiteURIFactory.php(222): CodeIgniter\HTTP\SiteURI->__construct(Object(Config\App), 'create/', NULL)
#3 C:\xampp\htdocs\machine_Test\system\HTTP\SiteURIFactory.php(43): CodeIgniter\HTTP\SiteURIFactory->createURIFromRoutePath('create/')
#4 C:\xampp\htdocs\machine_Test\system\Config\Services.php(798): CodeIgniter\HTTP\SiteURIFactory->createFromGlobals()
#5 C:\xampp\htdocs\machine_Test\system\Config\BaseService.php(258): CodeIgniter\Config\Services::uri(NULL, false)
#6 C:\xampp\htdocs\machine_Test\system\Config\BaseService.php(199): CodeIgniter\Config\BaseService::__callStatic('uri', Array)
#7 C:\xampp\htdocs\machine_Test\system\Config\Services.php(791): CodeIgniter\Config\BaseService::getSharedInstance('uri', NULL)
#8 C:\xampp\htdocs\machine_Test\system\Config\BaseService.php(258): CodeIgniter\Config\Services::uri()
#9 C:\xampp\htdocs\machine_Test\system\Config\Services.php(565): CodeIgniter\Config\BaseService::__callStatic('uri', Array)
#10 C:\xampp\htdocs\machine_Test\system\Config\Services.php(523): CodeIgniter\Config\Services::incomingrequest(Object(Config\App), false)
#11 C:\xampp\htdocs\machine_Test\system\Config\BaseService.php(258): CodeIgniter\Config\Services::request(NULL, false)
#12 C:\xampp\htdocs\machine_Test\system\Config\BaseService.php(199): CodeIgniter\Config\BaseService::__callStatic('request', Array)
#13 C:\xampp\htdocs\machine_Test\system\Config\Services.php(519): CodeIgniter\Config\BaseService::getSharedInstance('request', NULL)
#14 C:\xampp\htdocs\machine_Test\system\Config\BaseService.php(258): CodeIgniter\Config\Services::request()
#15 C:\xampp\htdocs\machine_Test\system\Debug\Exceptions.php(135): CodeIgniter\Config\BaseService::__callStatic('request', Array)
#16 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Exceptions\ConfigException))
#17 {main}
  thrown
【Previous Exception】
CodeIgniter\Exceptions\ConfigException
Config\App::$baseURL is invalid.
#0 C:\xampp\htdocs\machine_Test\system\HTTP\SiteURI.php(143): CodeIgniter\HTTP\SiteURI->normalizeBaseURL(Object(Config\App))
#1 C:\xampp\htdocs\machine_Test\system\HTTP\SiteURI.php(98): CodeIgniter\HTTP\SiteURI->determineBaseURL(Object(Config\App), NULL, NULL)
#2 C:\xampp\htdocs\machine_Test\system\HTTP\SiteURIFactory.php(222): CodeIgniter\HTTP\SiteURI->__construct(Object(Config\App), 'create/', NULL)
#3 C:\xampp\htdocs\machine_Test\system\HTTP\SiteURIFactory.php(43): CodeIgniter\HTTP\SiteURIFactory->createURIFromRoutePath('create/')
#4 C:\xampp\htdocs\machine_Test\system\Config\Services.php(798): CodeIgniter\HTTP\SiteURIFactory->createFromGlobals()
#5 C:\xampp\htdocs\machine_Test\system\Config\BaseService.php(258): CodeIgniter\Config\Services::uri(NULL, false)
#6 C:\xampp\htdocs\machine_Test\system\Config\BaseService.php(199): CodeIgniter\Config\BaseService::__callStatic('uri', Array)
#7 C:\xampp\htdocs\machine_Test\system\Config\Services.php(791): CodeIgniter\Config\BaseService::getSharedInstance('uri', NULL)
#8 C:\xampp\htdocs\machine_Test\system\Config\BaseService.php(258): CodeIgniter\Config\Services::uri()
#9 C:\xampp\htdocs\machine_Test\system\Config\Services.php(565): CodeIgniter\Config\BaseService::__callStatic('uri', Array)
#10 C:\xampp\htdocs\machine_Test\system\Config\Services.php(523): CodeIgniter\Config\Services::incomingrequest(Object(Config\App), false)
#11 C:\xampp\htdocs\machine_Test\system\Config\BaseService.php(258): CodeIgniter\Config\Services::request(Object(Config\App), false)
#12 C:\xampp\htdocs\machine_Test\system\Config\BaseService.php(199): CodeIgniter\Config\BaseService::__callStatic('request', Array)
#13 C:\xampp\htdocs\machine_Test\system\Config\Services.php(558): CodeIgniter\Config\BaseService::getSharedInstance('request', Object(Config\App))
#14 C:\xampp\htdocs\machine_Test\system\Config\BaseService.php(258): CodeIgniter\Config\Services::incomingrequest(Object(Config\App))
#15 C:\xampp\htdocs\machine_Test\system\Config\Services.php(538): CodeIgniter\Config\BaseService::__callStatic('incomingrequest', Array)
#16 C:\xampp\htdocs\machine_Test\system\Config\BaseService.php(258): CodeIgniter\Config\Services::createRequest(Object(Config\App))
#17 C:\xampp\htdocs\machine_Test\system\CodeIgniter.php(639): CodeIgniter\Config\BaseService::__callStatic('createRequest', Array)
#18 C:\xampp\htdocs\machine_Test\system\CodeIgniter.php(347): CodeIgniter\CodeIgniter->getRequestObject()
#19 C:\xampp\htdocs\machine_Test\public\index.php(79): CodeIgniter\CodeIgniter->run()
#20 C:\xampp\htdocs\machine_Test\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#21 {main}
in SYSTEMPATH\HTTP\SiteURI.php on line 197.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-07-15 21:15:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 21:16:43 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 21:17:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 21:18:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 21:22:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 21:28:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 21:29:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 21:29:38 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 21:29:38 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 21:29:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 21:29:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-07-15 21:33:08 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
